packages/app/src/messages/actions/delete-message.action.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import { MessagesEvent } from "App/messages/constants"
+import { deleteMessageRequest } from "App/messages/requests"
+import { DeleteMessageError } from "../errors/delete-message.action.error"
```

Comment:
😉 
```suggestion
import { DeleteMessageError } from "../errors/delete-message.error"
```

Reply:
Right - changed in https://github.com/mudita/mudita-center/pull/793/commits/b3335f33019d9cc0006efbab248bbcbb64d50f00